locale/en-us/intent/ask_llm.intent

```diff
@@ -1 +1 @@
-(ask|tell) (chatgpt|chat gpt|chat g p t|chat gpg|chat gptc|jack gpt|jack chi pt|fast chat|fastchat) {question}
\ No newline at end of file
+(ask|tell|task|half|sjet|asked|as) (chatgpt|chat gpt|chat g p t|chat gpg|chat gptc|jackgpt|jack gpt|jack chi pt|fast chat|fastchat|chat gbt|chatgbt|chat gpt4|chatgpt4|a language model|an llm) {question}
```

Comment:
Were these examples all observed on a Mark2?

`a language model` and `an llm` should have had the `a`/`an` normalized out before intent parsing (though its possible that those utterances are used here because of the way confidence is handled).

I'm curious what STT returned "sjet" and "jackgpt" because those don't appear to be valid English or likely typos

Reply:
Those were examples Richard gave me, I believe he was telling his Mark II "Ask ChatGPT..."